src/frontend/apps/sdk-consumer/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sdk-consumer",
   "private": true,
-  "version": "0.0.0",
```

Comment:
It is not needed to maintain this version as it's a local only package, as you want